python-avd/tests/pyavd/j2filters/test_hide_passwords.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import pytest
+from pyavd.j2filters.hide_passwords import hide_passwords
+
+VLAID_INPUT_HIDE_PASSWORDS = [
+    ("dummy", False, "dummy"),
+    ("dummy", True, "<removed>"),
+    ([1, 2, 3], True, "<removed>"),
```

Comment:
Value is defined to be string and here list is written.

Reply:
The PR is not ready for review, I am still working on some issue.
Then I will self-review once.